packages/block/examples/6110Requests.ts

```diff
@@ -17,26 +18,29 @@ const main = async () => {
   const depositRequestData = {
     pubkey: randomBytes(48),
     withdrawalCredentials: randomBytes(32),
-    amount: bytesToBigInt(randomBytes(8)),
+    amount: randomBytes(8),
     signature: randomBytes(96),
-    index: bytesToBigInt(randomBytes(8)),
+    index: randomBytes(8),
   }
-  const request = createDepositRequest(depositRequestData) as CLRequest<CLRequestType>
+  // flatten request bytes as per EIP-7685
```

Comment:
For the deposit request example, this should now also be edited because it does not match the spec anymore. Deploy the deposit contract, and then in a Prague block perform a deposit. The request should now contain the related LOG of the deposit data.